devtools/server/actors/replay/connection.js

```diff
@@ -228,23 +228,30 @@ Services.ppmm.addMessageListener("RecordReplayGeneratedSourceWithSourceMap", {
 
 Services.ppmm.addMessageListener("RecordingFinished", {
   async receiveMessage(msg) {
-    await sendCommand("Internal.setRecordingMetadata", {
+    // NOTE(dmiller): this can be null in the devtools tests for some reason, but not in production
+    // Not sure why.
```

Comment:
This is pretty strange, SendRecordingFinished in module.js always includes data when sending this message.  Catching this case is fine though, thanks!